File: launchdarkly/python/src/GetUserAttributeNamesExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.UsersBetaApi(api_client).get_user_attribute_names(
            project_key="projectKey_string",
            environment_key="environmentKey_string",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling UsersBetaApi#get_user_attribute_names: %s\n" % e)

File: launchdarkly/ruby/src/GetUserAttributeNamesExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::UsersBetaApi.new.get_user_attribute_names(
        nil, # project_key
        nil, # environment_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling UsersBetaApi#get_user_attribute_names: #{e}"
end
